crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -57,7 +57,7 @@ pub fn generate_cpp_properties(
     }
 
     generated.append(&mut generate_cpp_signals(
-        &signals,
+        &signals.iter().collect(), // Converting Vec<ParsedSignal> to Vec<&ParsedSignal>
```

Comment:
do we need the comment, as the others don't have the comment so maybe remove :-)